src/Form/FormLabel.js

```diff
@@ -0,0 +1,63 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import LabelBase from '../internal/LabelBase';
+
+export const styleSheet = createStyleSheet('FormLabel', () => {
+  return {
+    root: {}, // expected to be used
```

Comment:
That's a placeholder - looking/expecting some changes on general styling.  If not, we should rename `internal/LabelBase` to `Form/FormLabel`.